basis/io/directories/directories.factor

```diff
@@ -93,6 +85,18 @@ M: object copy-file
 : copy-files-into ( files to -- )
     '[ _ copy-file-into ] each ;
 
+! Moving and renaming files
+HOOK: move-file io-backend ( from to -- )
+
+M: object move-file ( from to -- )
+    [ normalize-path ] bi@ dupd copy-file delete-file ;
```

Comment:
Why not? This method is callable by user, to presumably it should sanitize/preprocess the parameters? Or is there a wrapper that does this?